mycore-base/src/main/resources/xsl/MyCoReWebPage-3.xsl

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE xsl:stylesheet [
+  <!ENTITY html-output SYSTEM "xsl/xsl-output-html.fragment">
+]>
+
+<!-- ============================================== -->
+<!-- $Revision: 1.25 $ $Date: 2007-12-06 16:17:45 $ -->
+<!-- ============================================== -->
+
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mcri18n="http://www.mycore.de/xslt/i18n"
+>
+  &html-output;
+
+  <xsl:include href="MyCoReLayout-3.xsl" />
+  <xsl:include href="xslInclude:MyCoReWebPage-3" />
+  <!-- <xsl:include href="xslInclude:MyCoReWebPage" /> -->
+
+  <xsl:variable name="PageID" select="/MyCoReWebPage/@id" />
+
+  <xsl:variable name="PageTitle">
+    <xsl:choose>
+      <xsl:when test="/MyCoReWebPage/section/@i18n">
+        <xsl:value-of select="mcri18n:translate(/MyCoReWebPage/section/@i18n)" />
+      </xsl:when>
+      <xsl:otherwise>
+        <xsl:choose>
+          <xsl:when test="/MyCoReWebPage/section[ lang($CurrentLang)]/@title != '' ">
```

Comment:
additional null/exists check or is `@title` always set

Reply:
i improved the conditions